proto/regen/ecocredit/v1/tx.proto

```diff
@@ -118,6 +118,29 @@ service Msg {
   // credit types to the network.
   // Since Revision 1
   rpc AddCreditType(MsgAddCreditType) returns (MsgAddCreditTypeResponse);
+
+  // AllowedClassCreator is a governance method that allows the addition of new
+  // addresses to the class creation allowlist.
+  // Since Revision 1
+  rpc AllowedClassCreator(MsgAllowedClassCreator)
+      returns (MsgAllowedClassCreatorResponse);
```

Comment:
I agree. There was also a previous request to update this terminology here to `AddRegistry` and `RemoveRegistry` with the list representing a list of allowed registries rather than "class creators". I will check again with Gregory and others but add and remove class creator should be ok for now.

Reply:
I agree. There was also a previous request to update this terminology here to `AddRegistry` and `RemoveRegistry` with the list representing a list of allowed registries rather than "class creators". I will check again with Gregory.

Reply:
I agree. There was also a previous request to update this terminology here to `AddRegistry` and `RemoveRegistry` with the list representing a list of allowed registries rather than "class creators". I will check again with Gregory. Add and remove class creator is ok for now but we might need to update before v5.0.